plugins/lsp/language-servers/default.nix

```diff
@@ -171,6 +171,12 @@ with lib; let
       package = pkgs.nodePackages.vscode-langservers-extracted;
       cmd = cfg: ["${cfg.package}/bin/vscode-html-language-server" "--stdio"];
     }
+    {
+      name = "java_language_server";
+      description = "Enable java language server";
+      package = pkgs.java-language-server;
+      cmd = cfg: ["${cfg.package}/bin/java-language-server"];
+    }
```

Comment:
```suggestion
    {
      name = "java-language-server";
      description = "Enable java language server";
      serverName = "java_language_server";
      cmd = cfg: ["${cfg.package}/bin/java-language-server"];
    }
```

Reply:
Nice, I was wondering how to get rid of the snake_case.